packages/backend/src/nest/registration/registration.service.ts

```diff
@@ -53,15 +40,26 @@ export class RegistrationService extends EventEmitter implements OnModuleInit {
     }
   }
 
-  public async registerOwnerCertificate(payload: RegisterOwnerCertificatePayload): Promise<void> {
+  public async registerOwnerCertificate(payload: RegisterOwnerCertificatePayload): Promise<string | undefined> {
```

Comment:
`string | undefined` is annoying type to work with

For functions that return stuff conditionally it's generally recommended to go with `string | null`